README.md

```diff
@@ -11,6 +11,17 @@
 
 For documentation, visit [https://frint.js.org](https://frint.js.org).
 
+Key characteristics and features include:
+
+* Gives your applications a **structure**
+* **Environment** agnostic (runs in browser, server, and CLI)
+* **Rendering** library agnostic (integrates with React, Vue, and Preact)
```

Comment:
I don't think you need comma after "Vue"

Reply:
@alexmiranda: Lord of grammar, please shine your light upon us :D

Reply:
@viacheslaff: seems optional: https://english.stackexchange.com/questions/30516/should-i-use-a-comma-before-and-or-or

Reply:
I didn't know that, thanks